src/ProxyClient/Varnish.php

```diff
@@ -109,7 +109,7 @@ public function invalidateTags(array $tags)
      */
     public function getTagsHeaderValue(array $tags)
     {
-        return implode(',', array_unique($this->escapeTags($tags)));
+        return array_unique($this->escapeTags($tags));
```

Comment:
but would you expect the psr to accept an array as header value and combine the array with commas? i would not expect that. and you keep calling escapeTags so i don't see what is changing here.